modules/new_profile/php/new_profile.class.inc

```diff
@@ -108,14 +110,25 @@ class NDB_Form_New_Profile extends NDB_Form
         // freeze it, just in case
         $this->form->freeze();
     }
+
     /**
-     * New profile function
+     * Does the setup required for this page. By default, sets up elements
+     * that are common to every type of page. May be overridden by a specific
+     * page or specific page type.
      *
-     * @return void
+     * @param string $name       The test name being accessed
+     * @param string $page       The subtest being accessed (may be null)
+     * @param string $identifier The identifier for the data to load on this page
+     * @param string $commentID  The CommentID to load the data for
+     * @param string $formname   The name to give this form
+     *
+     * @return none
      */
-    function new_profile()//@codingStandardsIgnoreLine
+    function _setupPage($name, $page, $identifier, $commentID, $formname)//@codingStandardsIgnoreLine
```

Comment:
You can remove the @codingStandardsIgnoreLine since the function name is no longer tied to the URL.